pypeman/contrib/http.py

```diff
@@ -143,11 +143,14 @@ class HttpRequest(nodes.BaseNode):
                        {'param1': 'omega', param2: ['alpha', 'beta']}
         :param client_cert: tuple with .crt and .key path
         :param binary: bool, Get response content as bytes
+        :param send_as_json: bool, If the method is a PATCH/POST/PUT, send data as json
+        :param json: bool, Parse Json response content
+        # TODO maybe add an auto parser if for example Content-Type header is application/json
     """
 
     def __init__(self, url, *args, method=None, headers=None, auth=None,
                  verify=True, params=None, client_cert=None, cookies=None,
-                 binary=False, json=False, **kwargs):
+                 binary=False, json=False, send_as_json=False, **kwargs):
```

Comment:
what is the difference between `json` which seems to be a bool and `send_as_json` which is a bool?

Reply:
Like i wrote in the docstring, send_as_json is for sending python data as json (asyncio will convert himself data, no need to have a Python2Json Node before) useful for POST/PUT/PATCH
json (maybe i had to rename the arg to receive_json ?) is for convert response content with a json.loads()